src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterFactory.cs

```diff
@@ -108,7 +112,15 @@ public virtual void Init(IProviderConfiguration providerCfg, string providerName
             adapterSettings = new EventHubStreamProviderSettings(providerName);
             adapterSettings.PopulateFromProviderConfig(providerConfig);
             hubSettings = adapterSettings.GetEventHubSettings(providerConfig, serviceProvider);
+#if NETSTANDARD
+            var connectionStringBuilder = new EventHubsConnectionStringBuilder(hubSettings.ConnectionString)
```

Comment:
I understand the need for these conditional computation blocks, but we should only use them if necessary.  ServiceBus libraries often add new capabilities prior to removing (deprecating) older ones.  With that in mind, it's possible that this pattern is already supported in the current nugets.  If it is, we can just move to the new pattern without the conditional code.  Please confirm that new patterns are not already supported in cases where we use conditional code.